README.md

```diff
@@ -1,12 +1,12 @@
-# Template Repository for Distroless Images
+# Template Repository for Chainguard Images
 
-This repository contains some basic files to spin up a new distroless image.
+This repository contains some basic files to spin up a new Chainguard image.
```

Comment:
If a person uses this template to create a repo in github.com/personal/repo, and builds an image from it, is that a Chainguard Image? I think we're saying no right? It has to be produced by and maintained by Chainguard to be one of those.

So if the audience is Chainguardians who want to make a new image (node, jre, etc.), we should spell that out better. But if the audience is anybody who wants to build an image _like_ we build our images, I think we should rephrase this. Maybe:

> This repository contains some basic files to spin up a new image in the style of a Chainguard Image.

WDYT?

Reply:
Yeah that's a good call-out. I updated the wording.